projects/plugins/jetpack/extensions/blocks/donations/deprecated/v2/index.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * WordPress dependencies
+ */
+import { RichText } from '@wordpress/block-editor';
+import { __, sprintf } from '@wordpress/i18n';
+import { createBlock } from '@wordpress/blocks';
+import formatCurrency from '@automattic/format-currency';
+
+/**
+ * Internal dependencies
+ */
+import { ANNUAL_DONATION, MONTHLY_DONATION, ONE_TIME_DONATION } from '../../common/constants';
+import { minimumTransactionAmountForCurrency } from '../../../../shared/currencies';
+
+const mapAttributes = oldAttributes => ( {
+	[ ONE_TIME_DONATION ]: oldAttributes[ ONE_TIME_DONATION ].show,
+	[ MONTHLY_DONATION ]: oldAttributes[ MONTHLY_DONATION ].show,
+	[ ANNUAL_DONATION ]: oldAttributes[ ANNUAL_DONATION ].show,
+	showCustomAmount: oldAttributes.showCustomAmount,
+	currency: oldAttributes.currency,
+	fallbackLinkUrl: oldAttributes.fallbackLinkUrl,
+} );
+
+const createDonationView = (
+	donationType,
+	donation,
+	chooseAmountText,
+	customAmountText,
+	currency
+) =>
+	createBlock(
+		'jetpack/donations-view',
+		{
+			type: donationType,
+		},
+		[
+			createBlock( 'core/heading', {
+				content: donation.heading || __( 'Make a one-time donation', 'jetpack' ),
+				level: 3,
+			} ),
+			createBlock( 'core/paragraph', {
+				content: chooseAmountText || __( 'Choose an amount', 'jetpack' ),
+			} ),
+			createBlock(
+				'core/buttons',
+				{
+					className: 'donations__amounts',
+				},
+				donation.amounts.map( ( amount, index ) =>
+					createBlock( 'jetpack/donations-amount', {
+						label: sprintf(
+							// translators: %d: Tier level e.g: "1", "2", "3"
+							__( 'Tier %d', 'jetpack' ),
+							index + 1
+						),
+						amount: Number( amount ),
+						currency,
+						baseAmountMultiplier: 2 * index,
```

Comment:
is this where [this problem](https://github.com/Automattic/jetpack/pull/22710/files#r806730506) is introduced? 

Multiplying the index by two will mean the baseAmountMultiplier gives 0, 2, 4 and then later on these values are multiplied by `minimumTransactionAmountForCurrency` (which for USD is 0.5)

Reply:
Yes, the problem must be where you pointed. I'll resolve this comment and we can continue with this issue on [the other comment](https://github.com/Automattic/jetpack/pull/22710/files#r806730506).